docusaurus/website/docusaurus.config.js

```diff
@@ -37,6 +37,14 @@ const config = {
   ],
 
   themeConfig: {
+    announcementBar: {
+      id: 'community_callout',
+      content:
+        'Learn more and connect with fellow plugin developers in the <a href="https://community.grafana.com/c/plugin-development/30" target="_blank" rel="noopener">Grafana Community Forum</a>. Ask questions, share knowledge, and get support from the Grafana team and community.',
```

Comment:
any way we can track the source (e.g. ?utm parameters?) to know how effective it is?

Reply:
Good point, I suspect we probably can. I'll take a look at how Discourse handles/surfaces this.